components/formats-gpl/src/loci/formats/in/CellWorxReader.java

```diff
@@ -335,7 +335,18 @@ else if (key.equals("XSites")) {
       }
       else if (key.equals("YSites")) {
         yFields = Integer.parseInt(value);
-        fieldMap = new boolean[yFields][xFields];
+        // if no site acquisition ("Sites" == "FALSE"),
+        // don't overwrite the single-site field map
+        if (fieldMap == null) {
```

Comment:
This assumes that the `Sites` key is always be defined before `XSites` in the examples right?

Reply:
Correct.  We're generally assuming that order matters when parsing the .HTD file; ```YSites``` must also be after ```XSites``` etc.  Not opposed to reworking that logic in a separate PR.